hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/spi/impl/OzoneManagerServiceProviderImpl.java

```diff
@@ -205,12 +205,12 @@ public void start() {
       LOG.error("Error staring Recon OM Metadata Manager.", ioEx);
     }
     long initialDelay = configuration.getTimeDuration(
-        RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY,
-        RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY_DEFAULT,
+        configuration.get(RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY),
+        configuration.get(RECON_OM_SNAPSHOT_TASK_INITIAL_DELAY_DEFAULT),
         TimeUnit.MILLISECONDS);
     long interval = configuration.getTimeDuration(
-        RECON_OM_SNAPSHOT_TASK_INTERVAL,
-        RECON_OM_SNAPSHOT_TASK_INTERVAL_DEFAULT,
```

Comment:
Given that we are setting the Recon OM snapshot interval value in MiniOzoneClusterImpl, this should just be conf.get(RECON_OM_SNAPSHOT_TASK_INTERVAL). We should not fall back to default.